audit-service/src/main/resources/application.yaml

```diff
@@ -50,8 +42,15 @@ management:
     shutdown:
       enabled: true
 
+  health:
```

Comment:
More of a question than an issue.  Noticed that we have some services that include the shutdown enabled as true for a graceful shutdown.  In others we don't.  Would this service be one that needs it?